src/coreclr/vm/amd64/excepamd64.cpp

```diff
@@ -367,7 +367,7 @@ RtlVirtualUnwind_Worker (
         // This is a jmp outside of the function, probably a tail call
         // to an import function.
         InEpilogue = TRUE;
-        NextByte += 2;
+        NextByte += 6;
```

Comment:
I am curious - how did you discover this issue?

Reply:
I only noticed that the code was wrong when I was trying to make sure that the new `jmp` instructions would be properly handled during unwinding. I haven't actually seen any issue (we have also never generated this particular form of `jmp` before this PR).